plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: plugins/katello/documentation
+title: Use Katello with Remote Databases
+version: nightly
+---
+
+# Setup Katello with remote databases
```

Comment:
General info here would be good about how you can externalize 1 or both databases. And that both databases can live on the same remote host but we recommend splitting them given MongoDB likes to own memory.